### PR TITLE
Fix: Remove PV root account from staking rewards destination options

### PR DESCRIPTION
**Pull request by egor0798**
Changes:
- Removed polkadot vault's root account from available options for staking rewards destination

### CODE REVIEW

File: src/renderer/pages/Staking/Operations/common/utils.tsx [hunk 1]
```diff
@@ -244,3 +245,9 @@ export const getExplorers = (address: Address, explorers: Explorer[] = []): [Inf
 
   return [{ items: explorersContent }];
 };
+
+export const filterPvRootAccounts = (accounts: Account[], wallets: Wallet[]) => {
+  const walletById = keyBy(wallets, 'id');
```
Comment (tuul-wq) [inline src/renderer/pages/Staking/Operations/common/utils.tsx:250]:
```suggestion
  const walletsMap = dictionary(wallets, 'id', walletUtils.isPolkadotVault);
```
This way you will have `true` or `false` for every wallet 😉

File: src/renderer/pages/Staking/Operations/common/utils.tsx [hunk 2]
```diff
@@ -244,3 +245,9 @@ export const getExplorers = (address: Address, explorers: Explorer[] = []): [Inf
...
+
+  return accounts.filter((a) => !accountUtils.isBaseAccount(a) || !walletUtils.isPolkadotVault(walletById[a.walletId]));
```
Comment (tuul-wq) [inline src/renderer/pages/Staking/Operations/common/utils.tsx:252]:
```suggestion
  return accounts.filter((a) => !accountUtils.isBaseAccount(a) || !walletsMap[a.walletId]);
```

File: src/renderer/pages/Staking/Operations/components/OperationForm/OperationForm.tsx [hunk 3]
```diff
@@ -80,13 +80,16 @@ export const OperationForm = ({
...
   const dbAccounts = useUnit(walletModel.$accounts);
+  const dbWallets = useUnit(walletModel.$wallets);
 
   const destinations = getDestinations(t);
 
   const [activePayout, setActivePayout] = useState<Address>('');
   const [payoutAccounts, setPayoutAccounts] = useState<ComboboxOption<Address>[]>([]);
 
-  const destAccounts = dbAccounts.filter((a) => accountUtils.isChainIdMatch(a, chainId));
+  const destAccounts = filterPvRootAccounts(dbAccounts, dbWallets).filter((a) =>
```
Comment (tuul-wq) [inline src/renderer/pages/Staking/Operations/components/OperationForm/OperationForm.tsx:90]: Let's create a whole function to get dest accounts (include `filterPvRootAccounts` logic into `getDestinationAccounts(wallets, accounts, chainId)`)

**Review by tuul-wq — COMMENTED** [review]
Small suggestion

**Review by Asmadek — APPROVED** [review]

**Pull request merged**